### DISCUSSION TITLE
execute SQL a same logic but not same sql have not same result

### DISCUSSION THREAD

**Discussion by linzhicong1996** [apache/doris#25606] [A - General / Q&A]
![image](https://github.com/apache/doris/assets/94167908/8f1db9be-392a-43a0-86da-02759e52affe)
![image](https://github.com/apache/doris/assets/94167908/8facafce-efe1-446b-b30e-08ce0182d5ff)
execute SQL a same logic but not same sql have not same result,i dont know why 'not in other table ' not have data

DORIS VERSION 2.0

**Comment by LemonLiTree** [comment]
Can you provide the table creation statements for these two tables?
Also, have you upgraded to 2.0?

**Reply by linzhicong1996** [reply]
![image](https://github.com/apache/doris/assets/94167908/7ad38219-1942-4a05-ad95-c7e02b383004)
yes，i used doris 2.0

**Reply by LemonLiTree** [reply]
My test on 2.0.2 is correct
<img width="1138" alt="image" src="https://github.com/apache/doris/assets/109049295/c1f0f590-72b8-460d-8814-63ab44eecac4">

**Comment by LemonLiTree** [comment]
Which version are you using?

**Reply by linzhicong1996** [reply]
![image](https://github.com/apache/doris/assets/94167908/b4e9ce57-2039-485e-8947-311be88388d3)

**Reply by LemonLiTree** [reply]
Try upgrading to 2.0.2
